src/SFML/Window/Unix/InputImpl.cpp

```diff
@@ -41,7 +41,16 @@ namespace priv
 bool InputImpl::isKeyPressed(Keyboard::Key key)
 {
     // Get the corresponding X11 keysym
-    KeySym keysym = 0;
+
+    if(key==Keyboard::CAPS){
+		Display* display = OpenDisplay();
+		unsigned int n;
+        XkbGetIndicatorState(display, XkbUseCoreKbd, &n);
+        CloseDisplay(display);
+        return (n&0x01)==1;
+	}
+
+    KeySym keysym;
```

Comment:
This results in an undefined `KeySym` for unhandled cases. Why not leave the default value of `0`?